webview/src/plots/components/comparisonTable/ComparisonTableCell.tsx

```diff
@@ -28,15 +31,25 @@ export const ComparisonTableCell: React.FC<ComparisonTableCellProps> = ({
   if (missing) {
     return (
       <div className={styles.noImageContent}>
-        <p>No Plot to Display.</p>
-        <RefreshButton
-          onClick={() =>
-            sendMessage({
-              payload: plot.revision,
-              type: MessageFromWebviewType.REFRESH_REVISION
-            })
-          }
-        />
+        {error ? (
+          <>
+            <ErrorTooltip error={error}>
+              <div>
+                <Error height={48} width={48} className={styles.errorIcon} />
+              </div>
+            </ErrorTooltip>
+            <RefreshButton
+              onClick={() =>
+                sendMessage({
+                  payload: plot.revision,
+                  type: MessageFromWebviewType.REFRESH_REVISION
+                })
+              }
+            />
+          </>
+        ) : (
+          <p className={styles.emptyIcon}>-</p>
```

Comment:
Why don't we keep `<p>No Plot to Display.</p>`?

Reply:
I stared at/reworded the statement for ~30 minutes then I decided that this is a table of data and that we could just display the same missing value that is displayed in the CLI experiments table. Could be the wrong decision but we can iterate with feedback 🙏🏻.